Documentation/DocuBlocks/Rest/Indexes/post_api_index_inverted.md

```diff
@@ -249,6 +249,27 @@ improve the performance of queries that return many documents.
 
 Default: `false`
 
+@RESTBODYPARAM{optimizeTopK,array,optional,string}
+An array of strings defining sort expressions that you want to optimize.
+This is also known as _WAND optimization_.
+
+If you query a collection with a `FILTER` operation accelerated by an
```

Comment:
It works only for `SORT some_score(d) DESC`. some_score exists only for `SEARCH` expression -> views (search-alias or arangosearch)

Reply:
So this property exist for inverted index but can be useful only for search-alias views (container of inverted indexes)

Reply:
Already addressed by https://github.com/arangodb/arangodb/pull/18690/commits/9f297dcb720348b80652eb141098292b7191ad63